test/core/card.spec.js

```diff
@@ -3,20 +3,15 @@ const expect = require('./../helper').expect;
 const Card = require('./../../src/core/card');
 
 describe('Card', () => {
-  beforeEach(() => {
-    this.card = new Card(2, 'spades');
-  });
+  const cardData = [2, 'spades'];
+  const card = new Card(...cardData);
```

Comment:
I fixed it moving variables' definition to `beforeEach` so they're defined before each test.